react-task/src/components/Register.js

```diff
@@ -0,0 +1,115 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+const Register = () => {
```

Comment:
Try keep the code neat by keep imports separate from the actual component code

```suggestion
import "bootstrap/dist/css/bootstrap.min.css";

const Register = () => {
```